template/partials/report.js

```diff
@@ -31,53 +34,98 @@ module.exports = function (results, hx) {
           <li><b>Latency average:</b> ${results.latency.average} ms</li>
         </ul>
       </div>
-      <div class='object latency'>
-        <div class='heading' onclick="growDiv(this)">
-        <h2 class='symbol'>-</h2>
-          <h2>Latency</h2>
-        </div>
-        <div class='content'>
-          <div class='measuringWrapper'>
-            <table class='table' style="width:100%">
-              <tr>
-                <th>Stat</th>
-                <th>Value</th>
-              </tr>
-              ${
-                Object.keys(results.latency).map((key) => {
-                  return hx`<tr>
-                    <td>${key}</td>
-                    <td>${results.latency[key]}</td>
-                  </tr>`
-                })
-              }
-            </table>
-          </div>
-        </div>
+      ${results['2xx'] + results.non2xx > 0 ? pie(results, hx) : ''}
+      ${results['2xx'] + results.non2xx > 0 ? panel1(results, hx) : panelWarn(results, hx)}
```

Comment:
What you could do is wrap all of the panel functions into their own `panels` function, and only call ```${results['2xx'] + results.non2xx > 0 ? panels(results, hx) : panelWarn(results, hx)}``` from the report body.

In the panels, you will want to ```return hx`${panel1(results, hx)}`..(all panels to show when no warning)..``` etc